ENH: Add linux-ppc64le support

matthewfeickert opened the pull request:
* Add linux-ppc64le build through cross-compile.
* Bump build number.

Attempting PR https://github.com/conda-forge/shellcheck-feedstock/pull/13 again.

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [N/A] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

matthewfeickert committed:
ENH: Add linux-ppc64le support

* Add linux-ppc64le build through cross-compile.
* Bump build number.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.
 * ℹ️ The recipe is not parsable by parser `conda-recipe-manager`. The recipe can only be automatically migrated to the new v1 format if it is parseable by conda-recipe-manager.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/24674093348](https://github.com/conda-forge/conda-forge-webservices/actions/runs/24674093348). Examine the logs at this URL for more detail.</sub>

h-vetinari commented:
You're going to need to change
https://github.com/conda-forge/shellcheck-feedstock/blob/719ee9cae92c145217a6dc4df8474a3f6e722716/recipe/meta.yaml#L9-L15

c.f. #9

isuruf committed:
Fix installing strip

isuruf committed:
Fix usin gstrip in build.sh

isuruf committed:
Fix using strip in bld.bat

bollwyvl commented:
Perhaps this is worth revisiting with the (more?) native GHA runner now?

matthewfeickert committed:
MNT: Re-rendered with conda-smithy 3.60.0 and conda-forge-pinning 2026.04.18.15.28.11

Other tools:
- conda-build 26.3.0
- rattler-build 0.62.2
- rattler-build-conda-compat 1.4.13

matthewfeickert commented:
> Perhaps this is worth revisiting with the (more?) native GHA runner now?

Done in the rebase. I left all of @isuruf's fixes so far in.

**edit**: Though I didn't address this comment https://github.com/conda-forge/shellcheck-feedstock/pull/16#issuecomment-3311863734, so that will probably need to happen.

bollwyvl commented on recipe/build.sh:
```diff
@@ -19,12 +19,12 @@ STACK_OPTS="\
 if [[ $target_platform =~ linux.* ]]; then
   install shellcheck "$PREFIX/bin/shellcheck"
-  strip --strip-all "$PREFIX/bin/shellcheck"
+  ${STRIP:-strip} --strip-all "$PREFIX/bin/shellcheck"
```
hm, no [upstream binary](https://github.com/koalaman/shellcheck/releases/tag/v0.11.0) for ppc64le....

matthewfeickert replied:
Yeah. https://github.com/koalaman/shellcheck/releases/tag/v0.11.0 So this would need to get built from source.

I currently have a grant proposal due this week and am traveling, so even though this might be low effort to chase down I will probably need to wait until next week to actually do more.

dbast replied:
There have been multiple attempts to build shellcheck from source on linux again... see other PRs like https://github.com/conda-forge/shellcheck-feedstock/pull/5 or https://github.com/conda-forge/shellcheck-feedstock/pull/9 ... though building on ppc64le would first require stack available for ppc64le https://github.com/conda-forge/stack-feedstock which is an upstream binary repack and there are no upstream stack binaries either https://github.com/commercialhaskell/stack/releases/tag/v3.9.3
